### ISSUE TITLE
FundsIn feature not working as expected

### ISSUE THREAD

**Issue by sri-miriyala** [mojaloop/project#529]
Priority : HIGHEST
Severity: HIGHEST

Env: ML AWS
API: Admin API
endpoint: POST /participants/{name}/account/{ID}

**Description:** 
After creating a DFSP, when I Deposit funds in their settlement account using the FundsIn feature, I can not see the funds in the settlement account. i.e. the funds are not deposited into the DFSP's settlement account.

**Expected behavior:**
1. Create a DFSP account with DFSP name and currency
2. Using FundsIn feature deposit funds into the settlement account
3. When you query the DFSP's account, I should see the deposited funds under that specific account of that DFSP.

**Actual behavior:**
1. Create a DFSP account with DFSP name and currency
2. Using FundsIn feature deposit ($5000) funds into the settlement account
3. When you query the DFSP's account, I did not see any funds deposited (showing "0") under that DFSP's currency account. **(FAILED)**

Screenshot:
![screen shot 2018-11-04 at 8 16 59 am](https://user-images.githubusercontent.com/38143240/47999933-09fb0580-e0d2-11e8-8ac1-90e18987a703.png)

Transfer ID not found in Central ledger DB:
![screen shot 2018-11-04 at 8 18 50 am](https://user-images.githubusercontent.com/38143240/47999968-30b93c00-e0d2-11e8-994c-459b66a00146.png)

Account 30, settlement account  is showing value "0"
![screen shot 2018-11-04 at 8 15 28 am](https://user-images.githubusercontent.com/38143240/48000003-475f9300-e0d2-11e8-8ae7-63f88af083b2.png)

Created By : Sri

**Comment by ggrg** [comment]
**Pull Requests**:
- [x] https://github.com/mojaloop/central-ledger/pull/202

**Comment by ggrg** [comment]
Hi @sri-miriyala - I've just retested everything and it seems to be working correctly for all operations: recordFundsIn, recordFundsOutPrepare, recordFundsOutCommit, recordFundsOutAbort & recordFundsOutPrepare with an amount that exceeds the allowed by the current balance of the settlement account. All checks (transfers) were created against dfsp1 and settlement account id 7.

**Comment by sri-miriyala** [comment]
@ggrg  thanks for the detailed note.

With new release v3.10, this feature is working as expected.
I did a transfer for $2000 into payerfsp account for FundsIn
I checked the payerfsp settlement acct balance
I checked the Hub reconciliation acct balance to see if $2000 fundsIn is reflected.
I checked the DB to see if that transfer is showing up for the amount.
<img width="793" alt="screen shot 2018-11-25 at 7 23 04 pm" src="https://user-images.githubusercontent.com/38143240/48987045-2bf01280-f0e9-11e8-8d4b-ed44020e28a4.png">
<img width="768" alt="fundsin 2000" src="https://user-images.githubusercontent.com/38143240/48987050-34484d80-f0e9-11e8-92ba-3f8eb66de367.png">
<img width="768" alt="payer acct bal after 2000 fundsin" src="https://user-images.githubusercontent.com/38143240/48987058-3ca08880-f0e9-11e8-80ed-2ce757e50211.png">
<img width="764" alt="hub recon after 2000 fundsin" src="https://user-images.githubusercontent.com/38143240/48987060-41fdd300-f0e9-11e8-8443-337d13ad1f1d.png">


Hence closing this issue.
